### ISSUE TITLE
Typo?

### ISSUE THREAD

**Issue by ChuyueSun** [dafny-lang/dafny#2981]
https://github.com/dafny-lang/dafny/blob/986e5c503dde977ab7c517b11ad22cec391c0fed/Source/DafnyCore/Verifier/Translator.cs#L860

**Comment by keyboardDrummer** [comment]
Yes this looks like it could be cleaned up. I'd be delighted if you create a pull request that does so. 

I will close this issue since it doesn't mention a Dafny user pain-point.

**Comment by davidcok** [comment]
Are you sure that one of the `sink`s was not supposed to be some other field?
